drv/sidecar-mainboard-controller/src/tofino2.rs

```diff
@@ -618,7 +732,7 @@ impl DebugPort {
         &self,
         segment: DirectBarSegment,
         offset: impl Into<u32> + Copy,
-        value: u32,
+        value: impl Into<u32> + AsBytes,
```

Comment:
```suggestion
        value: impl AsBytes,
```
(we don't actually use the `Into<u32>` bound)

Reply:
It does now, taking one of the u32 NewTypes and pulling out the bytes.